distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -681,27 +680,36 @@ public void printInactiveBundlesInfo() {
 
     private void printInactiveBundles(Consumer<String> headerConsumer,
             BiConsumer<String, Object[]> logConsumer) {
+        if (bundleService == null) {
```

Comment:
Those print statements are called fairly often during itest setup. I think the member variable makes sense to avoid having to retrieve the reference for every call. I don't think this class is touched very often, but since there are only two instances I am comfortable with leaving it explicit for now unless you feel strongly otherwise.